zds/forum/forms.py

```diff
@@ -50,8 +50,10 @@ def __init__(self, *args, **kwargs):
 
         self.helper.layout = Layout(
             Field('title', autocomplete='off'),
-            HTML('<div id="results" ><table id="tb-results" class="topics-entries"></table></div>'),
             Field('subtitle', autocomplete='off'),
+            HTML(u'<div id="topic-suggest" style="display:none;"><label>{0}</label>'
+                  '<div id="topic-result-container" data-neither="{1}"></div></div>'
```

Comment:
Erreur PEP-8 : `./zds/forum/forms.py:55:19: E127 continuation line over-indented for visual indent`